src/main/java/core/basesyntax/service/impl/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.OperationService;
+
+public class BalanceOperation implements OperationService {
+    @Override
+    public void operation(String line) {
```

Comment:
naming is confusing
1) method name should contain verb to get idea what will be done
2) BalanceOperation name looks like it should extend Operation but it extends OperationService
3) OperationService is very abstract and not giving idea about purpose (looks like basic CRUD service that should perform save/get/update/delete operations on Operation model)
4) Why method accepts String. This thing should accept operation
```suggestion
public class BalanceTransactionHandler implements TransactionHandler {
    @Override
    public void process(Transaction transaction) {
```

similar for other types of operations